packages/syft/src/syft/service/code/user_code.py

```diff
@@ -847,24 +869,177 @@ class UserCodeExecutionResult(SyftObject):
     result: Any
 
 
-def execute_byte_code(code_item: UserCode, kwargs: Dict[str, Any]) -> Any:
+def execute_byte_code(
+    code_item: UserCode, kwargs: Dict[str, Any], context: AuthedServiceContext
+) -> Any:
     stdout_ = sys.stdout
     stderr_ = sys.stderr
 
     try:
+        # stdlib
+        import builtins as __builtin__
+
+        original_print = __builtin__.print
+
+        class LocalDomainClient:
+            def __init__(self):
+                pass
+
+            def launch_job(self, func: UserCode, **kwargs):
```

Comment:
most of the juggling with new usercode and approval can be removed once @teo-milea finishes the UX for the new policy